requirements.txt

```diff
@@ -1,15 +1,9 @@
-shortuuid==0.4.3
 pystache==0.5.4
-configobj==5.0.6
 wikipedia==1.4.0
 requests==2.13.0
-pyOpenSSL==16.2.0
```

Comment:
Here's the original issue leading to upgrading the version listed in requirements: https://github.com/MycroftAI/mycroft-core/issues/705